### ISSUE TITLE
When running tests: Bundler could not find compatible versions for gem "activerecord"

### ISSUE THREAD

**Issue by dukedave** [composite-primary-keys/composite_primary_keys#398]
Following the example from the [Test](https://github.com/composite-primary-keys/composite_primary_keys#tests) section of the readme, I get:

```ruby
bundler install --without "mysql oracle sqlite sqlserver"
Updating https://github.com/rails-sqlserver/activerecord-sqlserver-adapter.git
Fetching gem metadata from https://rubygems.org/.......
Fetching version metadata from https://rubygems.org/..
Resolving dependencies...
Bundler could not find compatible versions for gem "activerecord":
  In Gemfile:
    activerecord-sqlserver-adapter (>= 0) ruby depends on
      activerecord (= 5.1.0.rc2) ruby

    activerecord (~> 5.0.0) ruby
```

Can others reproduce?

**Comment by dukedave** [comment]
Workaround: Delete the `group :sqlserver` block from `Gemfile`,

**Comment by cfis** [comment]
Yeah...its because the Gemfile was pointing to the activerecord-sqlserver-adapter repository (because the gem did not support active record 5.0 back when I did this).

Fixed in 0627b44.
